RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -2,7 +2,62 @@
 //  RockPaperScissors - main.swift
 //  Created by Baem, Dragon
 //  Copyright © yagom academy. All rights reserved.
-// 
+//
+
+startGame()
+
+func startGame() {
+    var winner = excuteScissorsRockPaper()
+    
+    if winner == "종료" {
+        return
+    }
+    
+    var gameResult = excuteMukChiBa(from: winner)
+    
+    while true {
+        switch gameResult {
+        case "무승부" :
+            print("\(winner)의 승리")
+            return
+        case "사용자" :
+            winner = "사용자"
+            gameResult = excuteMukChiBa(from: winner)
+        case "컴퓨터" :
+            winner = "컴퓨터"
+            gameResult = excuteMukChiBa(from: winner)
+        default :
```

Comment:
```suggestion
    while true {
        switch gameResult {
        case "무승부":
            print("\(winner)의 승리")
            return
        case "사용자":
            winner = "사용자"
            gameResult = excuteMukChiBa(from: winner)
        case "컴퓨터":
            winner = "컴퓨터"
            gameResult = excuteMukChiBa(from: winner)
        default:
```
보통 콜론 앞은 붙여주고 뒤는 띄어줍니다

Reply:
알겠습니다!~ 컨벤션 확인 감사합니다~~!